section4/exercises/ex3-2.rb

```diff
@@ -0,0 +1,32 @@
+class GoodDog
+  attr_accessor :name, :height, :weight
+
+  def initialize(n, h, w)
```

Comment:
You'll generally want to avoid using single letter variables in Ruby and most modern languages. The reason is readability for other devs, and it's simply not convention. In this case, I totally see where they came from and it makes sense enough, but definitely not a habit you want to get into.